CONTRIBUTING.md

```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
 
 You have been working on Centreon base code and want to submit it to us. Well... Again you are more than welcome and thank you in advance ! :clap:
 
-The pull request should respect the some requirements that can found in the following template [PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
+The pull request must comply with certain requirements which are set out in the following template
+[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
 
-> **Notice**: Any pull request that does not respect those requirements will be legitimately rejected !
+> **Notice**: A pull request which contains more than the described modifications or contains more than one issue will
+be rejected. Please take care of the description and open one pull request per issue.
+
+If the PR matches the expected requirements, it will be added to a refinement session with the dev team and the
+product owner.
+If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests
```

Comment:
`the PR will integrate the dev processes` -> `the PR will integrate our development workflow`